### PR TITLE
feat(j-s): CaseFile screen

### PR DESCRIPTION
**Pull request by oddsson**
# CaseFile screen

[Asana](https://app.asana.com/0/1199153462262248/1203046355378030/f)

## What

Implement the CaseFile screen in indictments

## Screenshots / Gifs

https://user-images.githubusercontent.com/3789875/196477465-8f289d81-0263-43ee-98b9-048d2dca662e.mov

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: apps/judicial-system/web/src/components/AccordionItems/IndictmentsCaseFilesAccordionItem/IndictmentsCaseFilesAccordionItem.gql.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { gql } from '@apollo/client'
```
[the remark this answers is not in the document]
Comment (Arnarkari93) [reply apps/judicial-system/web/src/components/AccordionItems/IndictmentsCaseFilesAccordionItem/IndictmentsCaseFilesAccordionItem.gql.ts:1]: I am not sure about it, I think it makes sense to have it next to the component. I think it should have the same name as the operation. In this case `UpdateFiles.gql.ts`

File: apps/judicial-system/web/src/components/AccordionItems/IndictmentsCaseFilesAccordionItem/IndictmentsCaseFilesAccordionItem.strings.ts [hunk 2]
```diff
@@ -0,0 +1,81 @@
+import { defineMessage } from 'react-intl'
+
+export const indictmentsCaseFilesAccordionItem = {
+  title: defineMessage({
```
Comment (gudjong) [inline apps/judicial-system/web/src/components/AccordionItems/IndictmentsCaseFilesAccordionItem/IndictmentsCaseFilesAccordionItem.strings.ts:4]: Wasn't the plan to use `defineMessages` on the top level?

File: apps/judicial-system/web/src/routes/Prosecutor/Indictments/CaseFile/CaseFile.strings.ts [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import { defineMessage } from 'react-intl'
+
+export const caseFile = {
+  heading: defineMessage({
```
Comment (gudjong) [inline apps/judicial-system/web/src/routes/Prosecutor/Indictments/CaseFile/CaseFile.strings.ts:4]: Consider using `defineStrings` on the top level.

File: apps/judicial-system/web/src/routes/Prosecutor/Indictments/Overview/Overview.tsx [hunk 4]
```diff
@@ -68,6 +74,8 @@ const Overview: React.FC = () => {
       activeSubSection={
         caseHasBeenSentToCourt
           ? undefined
+          : features.includes(Feature.CASE_FILE_ROUTE)
+          ? 4
```
Comment (gudjong) [inline apps/judicial-system/web/src/routes/Prosecutor/Indictments/Overview/Overview.tsx:78]: Consider using a defined constant instead of number.

**Review by gudjong — CHANGES_REQUESTED** [review]

File: apps/judicial-system/web/src/routes/Prosecutor/Indictments/Overview/Overview.tsx (hunk 4 above)
Comment (oddsson) [reply apps/judicial-system/web/src/routes/Prosecutor/Indictments/Overview/Overview.tsx:78]: This is just a temporary thing while we are using a feature flag and will be removed when we remove the flag.

**Pull request merged**